runtime/src/bank.rs

```diff
@@ -1374,8 +1375,9 @@ impl Bank {
             fee_structure: FeeStructure::default(),
         };
 
-        let total_accounts_stats = bank.get_total_accounts_stats().unwrap();
-        bank.store_accounts_data_len(total_accounts_stats.data_len as u64);
+        let accounts_data_len = bank.get_total_accounts_stats().unwrap().data_len as u64;
```

Comment:
For this code no, since it needs the bank to calculate the total accounts stats, so the `bank.get_total_accounts_stats()` then `bank.store_accounts_data_len()` cannot be removed. Luckily this version of new, `Bank::default_with_accounts()` is only called by tests (or if starting from genesis).

I'll answer the `Bank:: set_cost_tracker_account_data_size_limit()` question in the other comment.

Reply:
Done!